### DISCUSSION TITLE
modul anfrage

### DISCUSSION THREAD

**Discussion by Easton97-Jens** [ZoeyVid/NPMplus#556] [Q&A] [answered]
frage ist dises modul aktiv oder kann mann es mit rein nehem ngx_stream_realip_module

**Comment by Zoey2936** [comment]
Wurd im nächsten release hinzugefügt

**Reply by Easton97-Jens** [reply]
danke

**Reply by Easton97-Jens** [reply]
und wie sieht es mit dem ngx_stream_proxy_module modul aus da ich das auch brauche für mein vorhaben

**Reply by Zoey2936** [reply]
Ist dabei

**Reply by Zoey2936** [reply, marked as answer]
ist jetzt released
